idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/ui/di/DI.kt

```diff
@@ -0,0 +1,13 @@
+package io.github.composegears.valkyrie.ui.di
+
+import com.intellij.openapi.project.Project
+
+object DI {
```

Comment:
1) `DI : CoreModule by coreModule()` - less code in the usage section
2) No need to get platform module as it already part of core, just  line11 -> `project.provides {project}`

Reply:
discussed